src/Core/Components/Slider/FluentSlider.razor.cs

```diff
@@ -62,6 +72,15 @@ protected override void OnParametersSet()
         ArgumentNullException.ThrowIfNull(Step, nameof(Step));
     }
 
+    protected override async Task OnAfterRenderAsync(bool firstRender)
+    {
+        if (firstRender)
+        {
+            Module ??= await JSRuntime.InvokeAsync<IJSObjectReference>("import", JAVASCRIPT_FILE);
+        }
+        await Module!.InvokeVoidAsync("updateSlider", Element);
```

Comment:
Please NEVER use the `'!` to force the compiler to say that this object is not null. Technically, it is possible for `Module` to be `null`. So, prefer to use the test in this case: `if (Module is not null) ...`.

I suppose it's deliberate to execute this JavaScript method outside `firstRender'?

Reply:
@dvoituron

I got busy and  didn't have a chance to respond to this earlier. 

> Please NEVER use the '! to force the compiler to say that this object is not null. Technically, it is possible for Module to be null. So, prefer to use the test in this case: if (Module is not null) ....

It's my understanding that even though the compiler flags the possibility that `Module` may be null, it would require
1. `firstRender` is `false` on the firstpass which can't happen based on Blazor lifecycle.
2. Or, `InvokeAsync` returns null which it can't based on it's definition. If this call failed, an exception would be thrown.

I know this PR has been merged but can make this change for the sake of defensive programming 😄

> I suppose it's deliberate to execute this JavaScript method outside `firstRender'?

Yes, I did this in case `StateHasChanged` is called in the future for some reason.

Reply:
Hi @c0g1t8 @dvoituron , I hit this error today in Fluent v3.70,

Error: System.AggregateException: One or more errors occurred. (Value cannot be null. (Parameter 'jsObjectReference'))
 ---> System.ArgumentNullException: Value cannot be null. (Parameter 'jsObjectReference')
   at Microsoft.JSInterop.JSObjectReferenceExtensions.InvokeVoidAsync(IJSObjectReference jsObjectReference, String identifier, Object[] args)
   at Microsoft.Fast.Components.FluentUI.FluentSlider`1.OnAfterRenderAsync(Boolean firstRender) in /_/src/Core/Components/Slider/FluentSlider.razor.cs:line 81
   --- End of inner exception stack trace ---

Any idea?

Reply:
@geowong2022 

> I hit this error today in Fluent v3.70
 
Were you able to reproduce this error with the current release?  

BTW - If you can provide a repro, I'll take a look.